screens/bookings/booking/BookingSummary.tsx

```diff
@@ -27,21 +37,64 @@ const BookingSummary: React.FC<BookingSummaryProps> = ({
   navigation,
   route,
 }) => {
-  const bookingRequest = route.params.bookingRequest;
-  const parkingLot = route.params.parkingLot;
+  const { bookingDetailState, parkingLot } = route.params;
+  const { accessToken, authenticate } = useAuth();
+  const {
+    slotName,
+    checkInDate,
+    checkInTime,
+    checkOutDate,
+    checkOutTime,
+    specification,
+    price,
+    unit,
+  } = bookingDetailState;
+  const { mutateAsync: createBooking } = useCreateBooking();
   const [isOpenModal, setIsOpenModal] = useState(false);
-  const [isSendingRequest, setIsSendingRequest] = useState(false);
+  const [status, setStatus] = useState<CreatingBookingStatus>(
+    CreatingBookingStatus.PENDING
+  );
+  const createBookingRequest = formatCreateBookingRequest(
+    bookingDetailState,
+    parkingLot
+  );
+  // console.log("createBookingRequest", createBookingRequest);
+  const timeValidator = () => {
+    const status = validateTimeInputs(bookingDetailState);
+    switch (status) {
+      case ValidateStatus.SUCCESS:
+        return true;
+      case ValidateStatus.TIMEOUT:
+        Alert.alert("CheckIn or CheckOut timeout");
```

Comment:
It should be more informative e.g. `Alert.alert("Invalid booking time", "You should book at least xxx minutes before the booking time and checkout should be valid)`